vendor/modules.txt

```diff
@@ -459,6 +459,7 @@ github.com/go-ozzo/ozzo-validation/is
 github.com/go-stack/stack
 # github.com/go-stack/stack => github.com/go-stack/stack v1.8.0
 # github.com/godbus/dbus/v5 v5.0.3 => github.com/godbus/dbus/v5 v5.0.3
+## explicit
```

Comment:
Where does this come from?  Where are the vendor'ed files?

Reply:
As mentioned above, `github.com/godbus/dbus/v5` is already vendored because it's used by a transitive dependency. The files are already in vendor: https://github.com/kubernetes/kubernetes/tree/master/vendor/github.com/godbus/dbus/v5

The `#explicit` change was added as part of doing

```
$ hack/pin-dependency.sh github.com/godbus/dbus/v5 v5.0.3
$ hack/update-vendor.sh
```

I think it's added to signify this is now explicit direct dependency vs a transitive dependency.